integrations/facebook-pixel/lib/index.js

```diff
@@ -97,6 +99,7 @@ FacebookPixel.prototype.track = function(track) {
   var event = track.event();
   var revenue = formatRevenue(track.revenue());
   var whitelistPiiProperties = this.options.whitelistPiiProperties || [];
+  var customPiiProperties = this.options.blacklistPiiProperties || [];
```

Comment:
Any reason to call this `customPiiProperties` vs keeping it as `blacklistPiiProperties`? Confused me for a second.

Reply:
Yeah I chose it to differentiate between the `defaultPiiProperties` variable that exists in the `track` function where this setting is used. I figured custom/default was a better indication of what each was doing.